src/NextjsLambda.ts

```diff
@@ -90,6 +102,9 @@ export class NextJsLambda extends Construct {
     });
     this.lambdaFunction = fn;
 
+    // todo: once we figure out the correct S3 bucket, make sure permissions are appropriate.
+    props.cacheBucket.grantReadWrite(fn);
```

Comment:
In the spirit of least privilege, the server function should only need to read from cache, right? So this should be `grantRead`

Reply:
I'm not sure. I have to look more about what is actually writing to the cache. I'm slightly confused because I don't see `S3Cache` referenced anywhere, which in theory is what is writing to the cache. I'm not sure where the hook is here, who is actually writing to the cache. But I suspect it's the web server. I can figure this out I just am not sure at the moment.

Reply:
https://github.com/serverless-stack/open-next/blob/main/packages/open-next/src/adapters/server-adapter.ts#L204

We give nextjs a custom cache handler to get/set (read/write), so I believe we need both.

Reply:
OK, so I'll leave to read/write. One thing I'm not certainly about is if there is a reason this project is applying specific policies, e.g. https://github.com/jetbridge/cdk-nextjs/blob/main/src/NextjsS3EnvRewriter.ts#L80C1-L93C9 vs the `props.cacheBucket.grantReadWrite(fn);` syntax I used? Is there a preference for me to use the more verbose syntax?

Reply:
@kevin-mitchell, I think using `grantReadWrite` is ok.